### ISSUE TITLE
[Bug] getOption can't save parallelAxis selected state.

### ISSUE THREAD

**Issue by jarvis000000** [apache/echarts#18092]
### Version

5.4.1

### Link to Minimal Reproduction

https://codepen.io/jarvis000000/pen/jOpEoZE

### Steps to Reproduce

1. Changing parallelAxis selected states.
![image](https://user-images.githubusercontent.com/59681983/209038118-7e8ac372-7835-4c6e-804b-d07d80fecd82.png)

2. Click test button.Getting option of echart instance by getOption,and setting the option for the instance by setOption.
![image](https://user-images.githubusercontent.com/59681983/209038271-c04d8e6b-519d-48f9-8bdc-5366acd46724.png)

3. The parallelAxis selected state is cleared.
![image](https://user-images.githubusercontent.com/59681983/209038850-cc196fe2-658b-4a5d-ab86-33690ff3b366.png)

### Current Behavior

getOption can't save parallelAxis selected state.

### Expected Behavior

The option I get from getOption saved parallelAxis selected state,and I can recover the selected state by setting the option with setOption.

### Environment

```markdown
- OS: Windows 11
- Browser: Chrome 108.0.5359.125
- Framework:Vue@3
```

**Comment by helgasoft** [comment]
Parallel chart selections can be tracked with event [axisareaselected](https://echarts.apache.org/en/api.html#events.axisareaselected).
[Demo Code](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=PYBwLglsB2AEC8sDeBYAULWICGAnbANgQKYECCAHhAM4BcsA2uppkrACYQC29ADADSxo2LsXoByAAq4IAY2LjYAX37MWbTj1gBGQcNESAcsTCwA6sQgBzABZhFKtaw7d6AJj0ixscWS7AAV2h7ZVUMdScWTXoAZjCWFn1vcQBlWWBcBXiE2DAATxBk2WwwYisMvPFshPYS7HoGcQBRCnkiYmCqnwBxYGB2LvEAeQBpQYAhbAGAXUilNWns6mIZYjpkSPzCiRx8doIqyIIIaGIU_JJ6NgB3CHYwG3oAFlDI2rB6xkjMBm03ADoAJyAwTaXgCWAADg8PT6M2qLAYgKBIKhEIA7OjBMMxotvow3BC_hCAGwQ5qtUgkTqzcKYWmYeZoJQAbnQADc8LAANbEYggNloLh5ADCNjwYH-MAAFOJsFRqHhiNhliRZKUBoJpQBKBAAPg24WWqwQsGFYol_ysJgAsv1SDqrSYUisIGsdQxeNNBZgTpx5NRTca3U6wAAlbDXACS0H9a3GeTI6og7LOH1KsuwydT4m1PtgEAAZrBpX65Gt_tSrA89bxtZFefzTWWA_n0tBqMASJXgFZpY2QHn0Eo80A)

**Comment by jarvis000000** [comment]
@helgasoft  yes, I can get selections with [axisareaselected](https://echarts.apache.org/en/api.html#events.axisareaselected). But when I change series data with `setoption`,the selections will be cleared,and I can't find a aciton to recover the selections even I geted selections before.

**Comment by helgasoft** [comment]
The action to dispatch is [highlight](https://echarts.apache.org/en/api.html#action.highlight).
[Demo Code](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=PYBwLglsB2AEC8sDeBYAULWICGAnbANgQKYECCAHhAM4BcsA2uppkrACYQC29ADADSxo2LsXoByAAq4IAY2LjYAX37MWbTj1gBGQcNESAcsTCwA6sQgBzABZhFKtaw7d6AJj0ixscWS7AAV2h7ZVUMdScWTXoAZjCWFn1vcQBlWWBcBXiE2DAATxBk2WwwYisMvPFshPYS7HoGcQBRCnkiYmCqnwBxYGB2LvEAeQBpQYAhbAGAXUilNWns6mIZYjpkSPzCiRx8doIqyIIIaGIU_JJ6NgB3CHYwG3oANlDI4i4QG2xqGivIzGOp3OeUuyFu90eABZBKBsLIIPlaLpYOkCBlaOJMgN5uFMI5cRw6g1_oxtG4AHQATkpgm0vAEsAAHB4en0ZtUWAxKVSaUyGQB2fmCYZjRYkhhuBlkhlPBnNVqkEidWYElV49BKADc6AA1sRiCAEIxpprYOgwMArFYSEawLgAsRtWgAPTOlHHWQ62AwWDvcB5WDUHDyWCEGBWUNUai5YCwIIAWmWJFkYHQXDyAGEvrgwOSYAAKcTYKN4YjfUjEFPEAaCfMASgQAD4NuFlqsjems3hc1YTABZfqkevk3tgFIrCBresMXgmtQnTjyaOINuTkcmABK2GuAEloIu1uM8mQUxAAG5nMAlYiFuGQC_iOtOzAQABmsHzC7ka3JSqsD0bXg60iPUDSNL8l2fFEYGoYASF_S181AkAnw1VCXTdWQPS9H1sGgANAWIGNYBsawbGOWwwGddhgGuaAQAIbAAyTStSnYNNM2zXMC3ELC5B1LpXyCU8C12ERqAbVAMNgN9vXAKA4BoIRgFMWQvmgXt2FyGw1iIsBrljUtYCxIJamCdBXUwdJoGoUxQEgH1EFgTsuPXMAhnkgsGwst1nM47tyWWdzPOgfN7IU_g7QdYC0DUO8FMQC0rRtAB-WAACJSNsCi7HS2AJBouiGKY8QoIeGhyU4IMSjUk8HNCqSclyApvHimAOUwWorz3dhiAoehkI5JR0MwJLrSIxAAEIxpIJ1hs1IA)
![image](https://user-images.githubusercontent.com/13038071/209897801-ecc68c74-9d6f-4bf3-baf9-6b827f3d4eba.png)

**Comment by jarvis000000** [comment]
@helgasoft  
1. I see, but I not only want to recover highlight line but also need to recover axes selected areas.like this picture.
![image](https://user-images.githubusercontent.com/59681983/209918998-1da9315e-deac-4235-b4b2-034d9bd892c7.png)
Can the action which type is [brush](https://echarts.apache.org/en/api.html#action.brush) recover the axes selected areas?
2. Some times I want to update series data with `setOption`, is there any other way for do this and the axes selected areas aren't be cleared?

**Comment by helgasoft** [comment]
1. apparently selection is done only on **one vertical axis**. Once selected lines are set, it is easy to find the **selection ranges** (min/max) on all axes from the data itself.
2. when you _"update series data with setOption"_ the entire chart changes. I suppose you want to see which **new lines** pass thru **all** the **old** selection ranges? Then you have to first filter them from the data, like "New Weigth between 70 and 110 AND Amount between 77 and 82". Secondly - dispatch _highlight_ action on them by dataIndex.

**Comment by jarvis000000** [comment]
![image](https://user-images.githubusercontent.com/59681983/210027927-e0674beb-34fc-4d2c-a85f-6a473f6ccf9a.png)
This pictrue shows my case. When I change the filter params in **part 1** ，I hope **part3** can change selection ranges synchronously. And the data not in range still in **part3** with **downplay**.
But now it seems I can not change selection ranges in **part3** by action.  This is question 1.
For question 2 some times I have to change base data in **part2** , but I want to save the filter params in **part1**. And when I update series data with `setOption`, I losed the selection ranges in **part3** that should be saved in option with **getOption**. 
[*"Gets option object maintained in current instance, which contains configuration item and data merged from previous setOption operations by users, along with user interaction states. For example, switching of legend, zooming area of data zoom, and so on. Therefore, a new instance that is exactly the same can be recovered from this option."*](https://echarts.apache.org/en/api.html#echartsInstance.getOption)
I think they are operations by users and should be saved.
